modules/custom/az_metrics/az_metrics.module

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * @file
+ * Contains az_metrics.module.
+ */
+
+use Drupal\Component\Serialization\Json;
+
+const AZ_METRICS_URL = 'https://api.web.arizona.edu/v1';
```

Comment:
Is this privately controlled by Campus Web Services or something that could live in the Arizona Digital space?

Reply:
Last time I asked it sounded like we were leaning towards using a CWS domain since it was mostly for our team, but we can use a different domain if that's preferrable

Reply:
I agree that we should keep it with api.web.arizona.edu since the intent is primarily for our team.

Reply:
Yes, but... 
Why would anybody leave this module enabled if it sends data into a black box controlled by a different team?